txnkv/txnlock/lock_resolver.go

```diff
@@ -319,9 +331,20 @@ func (lr *LockResolver) ResolveLocks(bo *retry.Backoffer, callerStartTS uint64,
 // Read operations needn't wait for resolve secondary locks and can read through(the lock's transaction is committed
 // and its commitTS is less than or equal to callerStartTS) or ignore(the lock's transaction is rolled back or its minCommitTS is pushed) the lock .
 func (lr *LockResolver) ResolveLocksForRead(bo *retry.Backoffer, callerStartTS uint64, locks []*Lock, lite bool) (int64, []uint64 /* canIgnore */, []uint64 /* canAccess */, error) {
+	lr.saveResolvingLocks(locks, callerStartTS)
 	return lr.resolveLocks(bo, callerStartTS, locks, true, lite)
 }
 
+func (lr *LockResolver) saveResolvingLocks(locks []*Lock, callerStartTS uint64) {
```

Comment:
What if `ResolveLocks` happens concurrently for multiple regions?

For example, a batch get involves multiple regions and it encounters locks in more than one regions. But the `resolving` array always only includes the information of one region.